backend/database/seeders/TenantSeeder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+
+class TenantSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        DB::table('tenants')->insert([
+            'id' => 1,
+            'name' => 'test tenant1',
+            'email' => 'tenant1@test.com',
```

Comment:
```suggestion
            'email' => 'tenant1@example.com',
```

ref:
https://www.cotegg.com/blog/?p=2538
https://qiita.com/jnchito/items/198a2561a36c2c2ef5e3